tests/include/test/drivers/key_management.h

```diff
@@ -26,6 +26,8 @@ typedef struct {
     /* Count the amount of times one of the key management driver functions
      * is called. */
     unsigned long hits;
+    /* Subset of hits which only counts key operations with EC key */
+    unsigned long export_public_key_hits;
```

Comment:
Minor: `hits_export_public_key` would be more in line with the existing `aead.h` (the only that currently has systematic per-entry-point counters).

Reply:
Sure, I can fix that while we wait for the 2nd reviewer ;)